<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">background-color:gray</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1094</width>
         <height>536</height>
        </rect>
       </property>
       <widget class="NetworkView" name="networkView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>2000</width>
          <height>2000</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>2000</width>
          <height>2000</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExport_as_mininet_script"/>
    <addaction name="actionMetrik_data"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionController"/>
    <addaction name="actionSwitch"/>
    <addaction name="actionHost"/>
    <addaction name="actionLink"/>
    <addaction name="separator"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuShow_metric">
     <property name="title">
      <string>Show metric</string>
     </property>
     <addaction name="actionDelay"/>
     <addaction name="actionBandwidth"/>
     <addaction name="actionPLR"/>
    </widget>
    <addaction name="actionShow_ports"/>
    <addaction name="menuShow_metric"/>
   </widget>
   <widget class="QMenu" name="menuAlgorithms">
    <property name="title">
     <string>Algorithms</string>
    </property>
    <addaction name="actionGenetic_algorithm"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExport"/>
   <addaction name="menuTools"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAlgorithms"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGenetic_algorithm">
   <property name="text">
    <string>Genetic algorithm</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExport_as_mininet_script">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mininet Script</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionMetrik_data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Metric data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="actionController">
   <property name="text">
    <string>Controller</string>
   </property>
  </action>
  <action name="actionSwitch">
   <property name="text">
    <string>Switch</string>
   </property>
  </action>
  <action name="actionHost">
   <property name="text">
    <string>Host</string>
   </property>
  </action>
  <action name="actionLink">
   <property name="text">
    <string>Link</string>
   </property>
  </action>
  <action name="actionShow_ports">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show ports</string>
   </property>
  </action>
  <action name="actionDelay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delay</string>
   </property>
  </action>
  <action name="actionBandwidth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bandwidth</string>
   </property>
  </action>
  <action name="actionPLR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PLR</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NetworkView</class>
   <extends>QGraphicsView</extends>
   <header>networkview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
